tests/Doctrine/Tests/DBAL/ConnectionTest.php

```diff
@@ -861,8 +860,8 @@ public function testThrowsExceptionWhenInValidPlatformSpecified() : void
      */
     public function testRethrowsOriginalExceptionOnDeterminingPlatformWhenConnectingToNonExistentDatabase()
     {
-        /** @var VersionAwarePlatformDriverMock|MockObject $driverMock */
-        $driverMock = $this->createMock(VersionAwarePlatformDriverMock::class);
+        /** @var Driver|VersionAwarePlatformDriver|MockObject $driverMock */
+        $driverMock = $this->createMock([Driver::class, VersionAwarePlatformDriver::class]);
```

Comment:
This will have to be reworked due to https://github.com/sebastianbergmann/phpunit/issues/3955.